hnn_core/externals/lfp.py

```diff
@@ -0,0 +1,193 @@
+"""
```

Comment:
This file shouldn't be in externals. You keep stuff in externals when it needs to be synced with another repository. We have made extensive changes to the original code